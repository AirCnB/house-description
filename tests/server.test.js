const request = require('supertest');
const express = require('express');
const app = require('../server/app.js')


test('should respond to GET method with status code 200', () => {
  return request(app).get('/house/1').then( response => {
    expect(response.statusCode).toBe(200);
  });
});

test('should respond with json object', () => {
  return request(app).get('/house/1').then( response => {
    expect(response.type).toBe('application/json')
  });
});

test('pull correct object id from database', () => {
  return request(app).get('/house/1').then( response => {
    expect(response.body.id).toEqual(1);
  });
});