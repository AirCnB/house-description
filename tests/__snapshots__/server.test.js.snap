// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders main App 1`] = `
<div
  className="main-app"
>
  <DescriptionTitle
    info={Object {}}
  />
  <Highlights
    info={Object {}}
  />
  <DescriptionBody
    info={Object {}}
    show={false}
    toggleDescription={[Function]}
  />
  <Amenities
    info={Object {}}
    modal={false}
    togglePopup={[Function]}
  />
  <SleepingArrangements
    info={Object {}}
  />
  <Rules
    info={Object {}}
    showRules={false}
    toggleRules={[Function]}
  />
  <Cancellations
    showCancellation={false}
    toggleCancellation={[Function]}
  />
</div>
`;
